js/renovation/component_wrapper/common/editor.ts

```diff
@@ -32,14 +32,16 @@ export default class Editor extends Component {
         const $validationMessageWrapper = $('.dx-invalid-message.dx-overlay-wrapper');
         $validationMessageWrapper?.removeClass(INVALID_MESSAGE_AUTO);
 
+        const timeoutValue = 150;
```

Comment:
this constant name is too common to me. Maybe `timeToWaitBeforeBeforeShow` or smth like this?

Reply:
sounds good